.github/workflows/build_nightly.yml

```diff
@@ -296,6 +296,93 @@ jobs:
             _build/tests/cobol85/**/duration.txt
 
 
+  build_32_bit:
+    name: Build and test on 32-bit
+    needs: build
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    strategy:
+      fail-fast: true
+
+    steps:
+
+      - name: Get CI dist tarball
+        uses: actions/download-artifact@v4
+        with:
+          name: gnucobol-ci source distribution
+
+      - name: Extract CI dist tarball
+        run: tar -xvf gnucobol*.tar.* --strip-components=1
+
+      - name: Setup build environment
+        run: echo "TERM=vt100" >> $GITHUB_ENV
+
+      - name: Install packages
+        run: |
+          sudo dpkg --add-architecture i386
+          sudo apt -y update
+          sudo apt -y install make libtool gcc-9-multilib \
+                              gcc-9 libgmp-dev:i386 libncurses-dev:i386 \
+                              libdb5.3-dev:i386 libxml2-dev:i386 libcjson-dev:i386
```

Comment:
What about adding `libodbc2:i386 libodbcinst2:i386 unixodbc-dev:i386` (and potentially `--with-odbc` down there) - also in the existing GC4 one?